client/my-sites/plans-features-main/index.tsx

```diff
@@ -691,6 +692,15 @@ const PlansFeaturesMain = ( {
 		[]
 	);
 
+	const handlePlanTypeSelectorChange = useCallback( ( selectedItem: { url: string } ) => {
+		if ( hasQueryArg( selectedItem.url, 'intervalType' ) ) {
```

Comment:
`Note`:

DRY selectedItem.url

Reply:
No longer applicable as we refactor https://github.com/Automattic/wp-calypso/pull/86127#discussion_r1449253021